src/README.md

```diff
@@ -0,0 +1,227 @@
+<a name="readme-top"></a>
```

Comment:
- [ ] Nice job adding the README file, however, it's in the wrong position, kindly move it to the root directory and remove the previous README file from there

Reply:
Thank you, I will move it just now